### PR TITLE
Sequelize setup

### PR DESCRIPTION
**Pull request by karimabuseer**

### CODE REVIEW

**Review by senoodle — APPROVED** [review]

File: models/books.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      books.hasMany(models.user_books, {
+        foreignKey: 'id',
+        as: 'user_books_id',
+      });
+    }
+  };
+  books.init({
```
Comment (SipofTea) [inline models/books.js]: change to book

File: models/users_books.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      users_books.belongsTo(models.user, {
+        foreignKey: 'id',
+        as: 'user_id'
+      });
+      users_books.belongsTo(models.books, {
+        foreignKey: 'id',
+        as: 'books_id'
```
Comment (SipofTea) [inline models/users_books.js]: should be book_id

**Review by SipofTea — CHANGES_REQUESTED** [review]
Consistency with user vs books (singular vs plural) needs fixing

File: models/books.js (hunk 1 above)
Comment (karimabuseer) [reply models/books.js]: Done.

File: models/users_books.js (hunk 2 above)
Comment (karimabuseer) [reply models/users_books.js]: Done.

**Review by SipofTea — APPROVED** [review]

**Pull request merged**